.github/CODEOWNERS

```diff
@@ -7,6 +8,11 @@ content/conduct.adoc @jenkins-infra/board
 content/donate.adoc @jenkins-infra/board
 content/press.adoc @jenkins-infra/board
 
+# content/security - Security Team
```

Comment:
Perhaps it makes sense to make https://github.com/orgs/jenkins-infra/teams/security public and a code owner of this?

Reply:
Would be great, but I have no permissions to manage this team

Reply:
@jenkins-infra/security is now public, will add it